doc/changes/latest.inc

```diff
@@ -24,6 +24,8 @@ Current (1.0.dev0)
 Enhancements
 ~~~~~~~~~~~~
 
+- :func:`mne.time_frequency.tfr_array_multitaper` can now return results for ``output='phase'`` instead of an error (:gh:`10281` by `Mikołaj Magnuski`)
```

Comment:
Sphinx error is real:
```
changes/latest.inc:27: WARNING: py:obj reference target not found: Miko\u0142aj Magnuski
```
it needs to be
```suggestion
- :func:`mne.time_frequency.tfr_array_multitaper` can now return results for ``output='phase'`` instead of an error (:gh:`10281` by `Mikołaj Magnuski`_)
```